### ISSUE TITLE
Extra Blocks that are lying around.

### ISSUE THREAD

**Issue by Radical-from-Scratch** [NoMod-Programming/Whiskers-2.8#12]
Especially in control. There is no "increment counter" or "all at once" in Scratch. Once you finish the sound editor, I will fix this.

**Comment by NoMod-Programming** [comment]
I know, but these blocks are some "experimental blocks" left in the Scratch 2 code. They still work, so why no include them. But maybe I could have a workaround be saved for it (and the block be loaded from the workaround). Kinda like [this](https://scratch.mit.edu/discuss/topic/175369/?page=2#post-1714558)

**Comment by Radical-from-Scratch** [comment]
I have an idea. Make "hidden" block categories, so that if they're saved into a project they can still be loaded. Like the "say nothing" block.

**Comment by NoMod-Programming** [comment]
No. I still want it to look like Scratch 2... And have the blocks. I'll just keep them where they are for right now,

**Comment by Radical-from-Scratch** [comment]
Hm.....how about a "as close to 2.0 as we can" mode and a "With a little bit of shoe polish" mode
